app/uk.gov.hmrc.helptosaveproxy/config/CustomWSConfigParser.scala

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
I like the config structure followed here , if its not biggy, see if we stick to the same pattern 
`https://github.com/hmrc/soft-drinks-industry-levy-frontend/tree/master/app/sdil/config`

Reply:
if we do follow above, then i guess we can overcome the warnings (really ???)
```[warn] /Users/suresh/work/help-to-save-proxy/app/uk.gov.hmrc.helptosaveproxy/config/microserviceGlobal.scala:29: method current in object Play is deprecated: This is a static reference to application, use DI instead
[warn]   lazy val controllerConfigs: Config = Play.current.configuration.underlying.as[Config]("controllers")
[warn]                                             ^
[warn] there were four feature warnings; re-run with -feature for details
[warn] two warnings found
```

Reply:
I don't see these warnings, can you show me on Monday?